pkg/validate/rsync/validate/source_spec_validator.go

```diff
@@ -306,6 +306,41 @@ var InvalidSyncCode = "1061"
 
 var invalidSyncBuilder = status.NewErrorBuilder(InvalidSyncCode)
 
+// InvalidSecretName reports that a secret name is invalid.
+func InvalidSecretName(errs []string, secretName string) status.Error {
```

Comment:
Suggest passing in a `msg string` instead of `errs []string`. If the caller has a list of strings it can handle concatenation

Reply:
Done